package.json

```diff
@@ -36,11 +38,11 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
     "@babel/preset-env": "^7.6.3",
     "@fullhuman/postcss-purgecss": "^1.1.0",
-    "ajv": "^6.3.0",
     "autoprefixer": "^9.3.1",
     "babel-loader": "^8.0.0",
     "bootstrap": "^4.3.1",
     "clean-webpack-plugin": "^0.1.19",
+    "cross-spawn": "^7.0.2",
```

Comment:
@davidragsdale I don't think you need this anymore now that you revised the approach.

Reply:
```suggestion
```